CMakeLists.txt

```diff
@@ -19,6 +19,16 @@ function(travis_fixup exercise_dir alt_exercise_root)
 endfunction()
 
 option(EXERCISM_RUN_ALL_TESTS "Run all Exercism tests" On)
+option(EXERCISM_COMMON_CATCH "Link against a common Catch2 main lib." On)
+
+if(${EXERCISM_COMMON_CATCH})
```

Comment:
Is it worth removing the `${}` and changing this to:
```cmake
if(EXERCISM_COMMON_CATCH)
```
in the hopes of avoiding a future "gotcha"? It would match the others.

Reply:
👍